extension/js/content_scripts/webmail/gmail/gmail-element-replacer.ts

```diff
@@ -291,10 +291,12 @@ export class GmailElementReplacer extends WebmailElementReplacer {
   };
 
   private replaceActionsMenu = () => {
-    if ($('.action_menu_message_button').length <= 0) {
-      this.addMenuButton('reply', '#r');
-      this.addMenuButton('forward', '#r3');
-    }
+    $(this.sel.msgActionsBtn).on('click', () => {
+      if ($('.action_menu_message_button').length <= 0) {
+        this.addMenuButton('reply', '#r');
+        this.addMenuButton('forward', '#r3');
+      }
+    });
```

Comment:
this replacement will search for `this.sel.msgActionsBtn` in document DOM each second, which can lead to performance issues.

it'll be better to use `$(document).on('click', this.sel.msgActionsBtn, () => { ...` - it can be performed only 1 time and will catch all clicks at `this.sel.msgActionsBtn`.

Reply:
Thanks @sosnovsky - I've been finding a working alternative to this. You are right the statement that using $(document).on('click',.... but due to Gmail complexity, this isn't as simple as targeting the `this.sel.msgActionsBtn` but I highly agree with the concept your trying to teach to. I can't say it precisely why it doesn't work but after doing quite more debugging, I found a solution that is similar to what you've suggested. The result was significantly noticeable and love it showed up nicely without any delays with the recent changes.